src/pipx/interpreter.py

```diff
@@ -41,27 +43,76 @@ def __init__(self, source: str, version: str, wrap_message: bool = True):
                 message += "The provided version looks like a path, but no executable was found there."
             if potentially_pylauncher:
                 message += (
-                    "The provided version looks like a version for Python Launcher, " "but `py` was not found on PATH."
+                    "The provided version looks like a version, "
+                    "but both the python command and the Python Launcher were not found on PATH."
                 )
         if source == "the python-build-standalone project":
             message += "listed in https://github.com/indygreg/python-build-standalone/releases/latest."
         super().__init__(message, wrap_message)
 
 
+def find_unix_command_python(python_version: str) -> Optional[str]:
+    try:
+        parsed_python_version = version.parse(python_version)
+    except version.InvalidVersion:
+        logger.info(f"Invalid python version: {python_version}")
+        return None
+    else:
+        if (
+            parsed_python_version.epoch != 0
+            or parsed_python_version.is_devrelease
+            or parsed_python_version.is_postrelease
+            or parsed_python_version.is_prerelease
+        ):
+            logger.info(f"Unsupported python version: {python_version}")
+            return None
+
+        python_command_version = (
+            str(parsed_python_version.major)
+            if parsed_python_version.minor == 0
+            else f"{parsed_python_version.major}.{parsed_python_version.minor}"
+        )
+
+        python_command = f"python{python_command_version}"
+        if not shutil.which(python_command):
+            logger.info(f"Command `{python_command}` was not found")
+            return None
+
+        # Check consistency when micro part is specified
```

Comment:
I'd almost consider that as overshooting, actually...

Reply:
Log a warning, but use it anyway?

Reply:
Here we go! It looks much clear now. Thanks for the suggestion!